src/index.js

```diff
@@ -0,0 +1,74 @@
+import React from "react"
+import ReactDOM from "react-dom"
+import MyNotes from "./pages/myNotes"
+import reportWebVitals from "./reportWebVitals"
+import axios from "axios"
+import { createStore } from "redux"
+
+let NOTES = []
+let SHARED = []
+
+function authorizeUser(
+  state = { isAuthorized: false, notes: [], shared: [] },
+  action
+) {
+  switch (action.type) {
+    case "signIn":
+      return {
+        isAuthorized: !state.isAuthorized,
+        notes: action.payload.NOTES,
+        shared: action.payload.SHARED,
+      }
+    case "signOut":
+      return { isAuthorized: !state.isAuthorized }
+    default:
+      return state
+  }
+}
+
+let store = createStore(authorizeUser)
+
+function submitAutorization(values) {
+  axios
+    .get("https://mocki.io/v1/6e70ca5e-cb79-4b2f-8c99-8b99b08eb542")
+    .then((result) => {
+      let users = result.data
+      let userExists = false
+      for (let u = 0; u < users.length; u++) {
+        if (
+          users[u].email === values.email &&
+          users[u].password === values.password
+        ) {
+          userExists = true
+          NOTES = users[u].myNotes
+          SHARED = users[u].sharedNotes
+          store.dispatch({ type: "signIn", payload: { NOTES, SHARED } })
+          break
+        }
+      }
+      if (!userExists) {
+        alert("Invalid email or password")
+      }
+    })

```

Comment:
Move this function to a separate file in `src/api/auth.js`
And call it where neccessary